requests-for-comments/commnunity-2024.md

```diff
@@ -0,0 +1,47 @@
+Up to now, much of our thinking for the community section of the website has centered on creating a Hacker News-like experience where content drives the community. I think this is the wrong direction. This issue outlines why and an alternative vision of the community.
+
+## Why not the current direction?
+
+Content, both our own and good content curated _by us_ can still form an important part of our community, but:
+
+1. There are already numerous places to share content about building startups / engineering topics generally. Why should people post it to ours and not another, more established channel?  
```

Comment:
Not sure I agree with this. 
1. There are many for engineers, but I dont think any have a _product engineer_ focus
2. For building startups, which places exist to share content?

Reply:
1. I think that's a narrow distinction we care about more than most people do. 
2. Numerous sub-reddits, Slack groups, Discords, Indiehackers. That said, most tend to discourage sharing "content" because it's just an invitation to spam them.